datatypes/src/primitives/spatial_partition.rs

```diff
@@ -13,7 +13,7 @@ use super::SpatialResolution;
 
 /// Common trait for axis-parallel boxes
 pub trait AxisAlignedRectangle: Copy {
-    /// create a new instance defined by min and max coordinate values
+    /// create a new instance defined by `min` (lower left) and `max` (upper right) coordinate
```

Comment:
Is `from_min_max` the minimum enclosing rectangle for coordinates a and b or is the max taken from b and min from a?